content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,359 @@
+=====================
+Payroll configuration
+=====================
+
+Odoo *Payroll* is used to process work entries and create payslips for employees, including both
+regular pay and commission. Payroll works in conjunction with other Odoo apps, such as *Employees*,
+*Timesheets*, *Time Off*, and *Attendances*.
+
+The *Payroll* app helps ensure there are no issues or conflicts when validating work entries,
+handles country-specific localizations to ensure that payslips follow local rules and taxes, and
+allows for salary assignments. Payroll configuration is critical to ensure accurate and timely
+processing of payslips.
+
+Settings
+========
+
+The :guilabel:`Settings` section is where localization settings are configured. *Localizations* are
+country-specific settings pre-configured in Odoo at the creation of the database, and account for
+all taxes, fees, and allowances for that particular country.  The localization section of the
+settings includes a detailed view of all benefits provided to employees.
+
+To access the settings, go to :menuselection:`Payroll --> Configuration --> Settings`. Whether or
+not payslips are posted in accounting, and whether SEPA payments are created, is selected here.
+
+.. image:: configuration/payroll-settings.png
+   :align: center
+   :alt: Settings available for payroll.
+
+Any country-specific localizations are set up in the  :guilabel:`Localization` section of the
+settings screen. All localization items are pre-populated when the country is specified during the
+creation of the database. It is not recommended to alter the localization settings unless
+specifically required.
+
+Work entries
+============
+
+A *work entry* is an individual record on an employee's timesheet. Work entries can be configured to
+account for all types of work and time off, such as `Attendance`, `Sick Time Off`, `Training`, or
+`Public Holiday`.
```

Comment:
I also agree- I was thinking since the terms are also generic, like sick time off, I thought I could leave them lower case (I feel like I use WAY too many GUI labels in my docs). =)